certified-connectors/PDFcross/apiDefinition.swagger.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "PDFcross is a service for manipulating PDF files, such as merging PDF files into one file.",
```

Comment:
@nabe-hiromichi We have many PDF-related questions, It would not be easy to find your connector in the search. Can you please provide more description of your connector should describe concisely the main purpose and value offered by your connector.

Docs - https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector

Reply:
I changed description. Please check it.